src/IW/Db/Issue.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+-- | SQL queries to work with the @users@ table.
+
+module IW.Db.Issue
+       ( getIssues
+       , getIssueById
+       , getIssuesByLabel
+       ) where
+
+import IW.App (WithError)
+import IW.Core.Issue (Issue)
+import IW.Db.Functions (WithDb, asSingleRow, query, queryRaw)
+
+getIssues :: (WithDb env m, WithError m) => m [Issue]
+getIssues = queryRaw [sql|
+    SELECT *
```

Comment:
`SELECT *` is never a good idea. Let's be implicit instead :slightly_smiling_face:

Reply:
Got it.